scripts/utils/migrate_mcumgr_kconfigs.py

```diff
@@ -82,7 +82,7 @@
     "MCUMGR_BUF_SIZE" : "MCUMGR_TRANSPORT_NETBUF_SIZE",
     "MCUMGR_BUF_USER_DATA_SIZE" : "MCUMGR_TRANSPORT_NETBUF_USER_DATA_SIZE",
     "MCUMGR_SMP_BT" : "MCUMGR_TRANSPORT_BT",
-    "MCUMGR_SMP_BT_AUTHEN" : "MCUMGR_TRANSPORT_BT_AUTHEN",
+    "MCUMGR_SMP_BT_AUTHEN" : "MCUMGR_TRANSPORT_BT_PERM_RW_AUTHEN",
```

Comment:
this needs another entry to change MCUMGR_TRANSPORT_BT_AUTHEN to MCUMGR_TRANSPORT_BT_PERM_RW_AUTHEN, also please add note in migration guide

Reply:
thanks, done